source/__tests__/integration/home-page.test.js

```diff
@@ -21,15 +21,33 @@ describe('Sinopia Profile Editor Homepage', () => {
   })
 
   describe('header', () => {
-    it('title', async() => {
-      expect.assertions(2)
-      await pupExpect.expectSelTextContentToBe('h2.sinopia-subtitle > a:nth-child(1)', 'Sinopia')
```

Comment:
for unknown reasons, after I made the changes to `index.html`, jest became unhappy with these selectors for header links.  🤷‍♀ 

I am never fond of "nth-child" links if there is something more descriptive, in any case.